blank-canvas/inc/block-patterns.php

```diff
@@ -18,7 +18,7 @@ function blank_canvas_register_block_patterns() {
 
 			register_block_pattern_category(
 				'blank-canvas',
-				array( 'label' => __( 'Blank Canvas', 'blank-canvas' ) )
+				array( 'label' => _x( 'Blank Canvas', 'theme name', 'blank-canvas' ) )
```

Comment:
Should the context not be `block pattern name`? It does collide with the theme name (which doesn't get a context since it's extracted from style.css) but this here is the name of a block pattern, or is it not? If this is supposed to be the theme name here, then actually no change is necessary.

Reply:
Considering the context, I think it's the theme name used a category label that groups various block patterns and therefore the context `theme name` suggests that it shouldn't be translated. Though, I might be wrong so I'll be happy to hear other's thoughts.

Reply:
Are block pattern category names usually translated? If it shouldn't be translated since it's matching the theme name, wouldn't it be easier to just not wrap it in the gettext call?

Reply:
@dlind1, I'm not sure if or in what case could be translated, but either way if the category name refers to the theme, we should **not** remove the gettext call they both keep matching.